actions/class.TestRunner.php

```diff
@@ -840,49 +870,50 @@ public function comment() {
             $transmitter = new taoQtiCommon_helpers_ResultTransmitter($resultStore);
             $transmitter->transmitItemVariable($variable, $transmissionId, $itemUri, $testUri);
         }
-	}
+    }
 
-	/**
-	 * Retrieve the Test Definition the test session is built
-	 * from as an AssessmentTest object. This method
-	 * also retrieves the compilation directory.
-	 *
+    /**
+     * Retrieve the Test Definition the test session is built
+     * from as an AssessmentTest object. This method
+     * also retrieves the compilation directory.
+     *
      * @param string $qtiTestCompilation (e.g. <i>'http://sample/first.rdf#i14363448108243883-|http://sample/first.rdf#i14363448109065884+'</i>)
      *
-	 * @return AssessmentTest The AssessmentTest object the current test session is built from.
-	 */
-	protected function retrieveTestDefinition($qtiTestCompilation) {
+     * @return AssessmentTest The AssessmentTest object the current test session is built from.
+     */
+    protected function retrieveTestDefinition($qtiTestCompilation)
+    {
         $directoryIds = explode('|', $qtiTestCompilation);
-	    $directories = array(
+        $directories = [
             'private' => $this->getDirectory($directoryIds[0]),
             'public' => $this->getDirectory($directoryIds[1])
-        );
+        ];
 
-	    $this->setCompilationDirectory($directories);
-	    $testDefinition = \taoQtiTest_helpers_Utils::getTestDefinition($qtiTestCompilation);
-	    $this->setTestDefinition($testDefinition);
-	}
-
-	/**
-	 * Retrieve the current test session as an AssessmentTestSession object from
-	 * persistent storage.
-	 *
-	 */
-	protected function retrieveTestSession() {
-	    $qtiStorage = $this->getStorage();
-	    $sessionId = $this->getServiceCallId();
-
-	    if ($qtiStorage->exists($sessionId) === false) {
-	        common_Logger::t("Instantiating QTI Assessment Test Session");
+        $this->setCompilationDirectory($directories);
+        $testDefinition = \taoQtiTest_helpers_Utils::getTestDefinition($qtiTestCompilation);
```

Comment:
Add this to use section

```suggestion
        $testDefinition = taoQtiTest_helpers_Utils::getTestDefinition($qtiTestCompilation);
```